micro/example_micro_handler_test.go

```diff
@@ -38,7 +38,7 @@ func (g *Greeter) Hello(ctx context.Context, name *string, msg *string) error {
 
 func ExampleNewHandlerWrapper() {
 	//Use gRPC reporter for production
-	r, err := reporter.NewLogReporter()
+	r, err := reporter.NewGRPCReporter("skywalking-oap:11800")
```

Comment:
@zaunist Hi, I think the example should use `LogReporter`, so it was good before :)

Reply:
Agree，I already update it